test/integration/ads_integration_test.cc

```diff
@@ -790,10 +790,6 @@ TEST_P(AdsIntegrationTest, NodeMessage) {
   }
   EXPECT_THAT(node->user_agent_build_version(), ProtoEq(VersionInfo::buildVersion()));
   EXPECT_GE(node->extensions().size(), 0);
-  // Check that all extensions are enabled and version the same as the user-agent
-  for (const auto& ext : node->extensions()) {
```

Comment:
Have we lost functional coverage with this removal?

Reply:
Not really. This is covered by the registry_test.cc.
Coverage test picks up all of the factories registered in the unit tests and the assumption that this loop only looks at the default envoy extensions is incorrect.